### PR TITLE
Beta

### PR DESCRIPTION
**Pull request by wojtek-krysiak**

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+node_modules
```
Comment (wojtek-krysiak) [inline .npmignore:1]: these folders wont go to NPM

File: .npmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+access=public
```
Comment (wojtek-krysiak) [inline .npmrc:1]: this is required whe you publish @admin-bro packages (@ - npm organization)

File: example-app/cypress.env.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "MONGO_URL": "mongodb://localhost:27017/koa-example",
```
Comment (wojtek-krysiak) [inline example-app/cypress.env.json:2]: I moved the envs so they can be used by both cypress and the example app

File: example-app/cypress/integration/auth.spec.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
+context('SignIn page', () => {
+  beforeEach(() => {
+    cy.login()
+  })
+
+  it('logs in to the application', () => {
+    cy.location('pathname').should('not.include', 'admin/login')
```
Comment (wojtek-krysiak) [inline example-app/cypress/integration/auth.spec.ts:7]: one simple login e2e test

File: package.json [hunk 5]
```diff
@@ -39,13 +40,15 @@
     "semantic-release-slack-bot": "^1.6.2",
     "typescript": "^3.9.7"
   },
-  "dependencies": {
-    "koa2-formidable": "^1.0.2",
+  "peerDependencies": {
```
Comment (wojtek-krysiak) [inline package.json:43]: I moved admin-bro, koa, formidable and router to peerDependency - the rule if thumb is to move there everything that you export from your library.

**Review by kamil-kolenda — APPROVED** [review]
LGTM

**Pull request merged**